### PR TITLE
Translated Instabots and cybersecurity

### LINKED ISSUE #39: [Translations] Instabots and cybersecurity - fi
**What event the translation is missing for:**
[Instabots and cybersecurity ](https://github.com/turkupy/turkupy.github.io/blob/dev/content/events/fi/instabots-and-cybersecurity/index.md)

### PR DESCRIPTION
**Pull request by LehtinenHenna**
Resolves #39 .

**Why this pull request exists?**
A translation was needed for the event Instabots and Cybersecurity
**Changes made in this pull request:**
- I translated the text from English to Finnish.

@turkupy/turkypy-maintainers

### CODE REVIEW

File: content/events/fi/instabots-and-cybersecurity/index.md [hunk 1]
```diff
@@ -9,12 +9,12 @@ type: "event"
 lang: fi
 ---
 
-In January we visited Identio for a meetup about Instagram bots and Cybersecurity.
+Tammikuussa kokoonnuimme Idention tiloihin puhumaan Instagramista ja kyberturvallisuudesta.
 
-Heta Pärssinen is a fullstack developer working at Duunitori. Building bots for Instagram has been her occasional hobby and she wanted to share the technical secrets with us at this meetup.
+Heta Pärssinen työskentelee fullstack -kehittäjänä Duunitorilla. Hän on harrastuksekseen rakentanut Instagram botteja, ja halusi jakaa miitissämme niiden teknisiä salaisuuksia.
 
 ![Heta presenting about Instabots.](heta.jpg)
```
Comment (eevajonnapanula) [inline content/events/fi/instabots-and-cybersecurity/index.md:16]: A small note: These texts are alt-texts for the images, so could you also translate them? 😊

**Review by eevajonnapanula — CHANGES_REQUESTED** [review]
Looking really good! There were two alt tags missing translations (see the comment in review) so if you could add them too, it would be awesome! 😊

File: content/events/fi/instabots-and-cybersecurity/index.md (hunk 1 above)
Comment (LehtinenHenna) [reply content/events/fi/instabots-and-cybersecurity/index.md]: Hi, I translated now the texts that were in square brackets but should I make a new pull request now or how are the new changes transferred to the file?

Comment (eevajonnapanula) [reply content/events/fi/instabots-and-cybersecurity/index.md]: They have been automatically pushed to this PR, so no need to make anything more! 😊

**Comment by eevajonnapanula** [comment]
Thank you for your contribution 🎉

**Pull request merged**